users_ldap_populate/models/users_ldap.py

```diff
@@ -85,7 +85,7 @@ def action_populate(self):
                         )
                         user_id = res[0]
                     else:
-                        raise UserError(
+                        raise UserError from None(
```

Comment:
You can capture the exception in `except AccessDenied as e:` and launch this one from there.